certificateshop-logic/src/main/java/com/epam/esm/dao/impl/jpa/JpaUserDaoImpl.java

```diff
@@ -0,0 +1,186 @@
+package com.epam.esm.dao.impl.jpa;
```

Comment:
Why do you use native query in almost all repository methods? There are more easier approaches to update or create entities.

Reply:
Because other approaches send the queru to infinit loop. That is the point. My architecture is so. It can be upgraded in next releases potentially, but now it works the same in 'jdbc' and 'jpa' profiles. this desigion was made at the beginning of the implementation and only the one way was found to get it work properly according to the architecture of the app.

Reply:
To have 'orders' by 'user' let users to make an order simultaneously with creating account (creating new user in the application). It is the one of the argumenst why this decision was made.